TypeCobol/Compiler/SqlScanner/SqlToken.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using TypeCobol.Compiler.Scanner;
+
+namespace TypeCobol.Compiler.SqlScanner
+{
+    public class SqlToken : Token
```

Comment:
Do we need a new type of Token ? What about existing variants like ContinuationToken, TokenGroup, etc. Creating a new type of token for SQL somehow breaks the hierarchy and functionality of token classes when in SQL. Is SQL subject to REPLACE mechanism and continuation rules ?

Reply:
Any token shall be created by a scanner process, so create a SqlToken type that inherits from Token should not break anything, even at the token is produce by a preprocessing mechanism. But I agree no use to specialize here let's use SqlFamily for that.

Reply:
You are right I didn't use it :-|

Reply:
done in 5a26682